.github/workflows/build_msi_installer.yml

```diff
@@ -74,7 +74,7 @@ jobs:
             if ($version -ne $run_version) {
               throw "Version input does not match the version in promptflow package. Version input: $version, version in promptflow package: $run_version"
             }
-          } 
+          }
```

Comment:
I think we  also need update **- name: Install stable promptflow** when all packages are released in pypi